src/app/util/attribute-storage-helpers.h

```diff
@@ -0,0 +1,75 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ * Helpers for manipulating values that are going into and out of the attribute
```

Comment:
any chance we can avoid 'helpers' as description? 

- consider dropping extra 'file' descriptions as they generally just restate the file name (the name here already says 'helpers for attribute storage')

- could we get by without saying 'helpers'? Based on current content, this contains 'numeric-attribute-traits' so maybe that should be in the name and that would both make it more obvious to users what to include and what is included as well as not allowing this file to start containing unrelated bits of code because  'helpers' can be anything.

Reply:
I can certainly drop this comment.

As far as what goes in here, there are some string and float related things that will need to get added too.  I was just trying to not scope-creep this PR too much; the string bits will need more of a refactor of the code in ember-compatibility.

That said, I could name this attribute-storage-null-handling.h.  Thoughts @andy31415 ?

Reply:
That seems reasonable - i would understand than that this is needed for null support.

Reply:
Done.